src/xcode/ENA/ENAUITests/ENAUITests.swift

```diff
@@ -15,6 +15,7 @@ class ENAUITests: XCTestCase {
 		setupSnapshot(app)
 		app.setDefaults()
 		app.launchEnvironment["IsOnboarded"] = "NO"
+		app.launchArguments.append(contentsOf: ["-userNeedsToBeInformedAboutHowRiskDetectionWorks", "NO"])
```

Comment:
Funny that we have this LaunchArgument, but didn't use it 👍

Reply:
Coincidentally I saw that name today :)